src/main/java/vazkii/botania/common/item/lens/ItemLens.java

```diff
@@ -319,6 +319,6 @@ public ItemStack setCompositeLens(ItemStack sourceLens, ItemStack compositeLens)
 
 	@Override
 	public boolean shouldPull(ItemStack stack) {
-		return stack.getItemDamage() != STORM;
+		return stack.getItemDamage() < 5000;
```

Comment:
I was initially going to use a beam of mana bursts, but then I found that bad things happened if you directed it towards a mana acceptor.